minifi_main/MiNiFiMain.cpp

```diff
@@ -263,6 +263,7 @@ int main(int argc, char **argv) {
     configure->setHome(minifiHome);
     configure->loadConfigureFile(DEFAULT_NIFI_PROPERTIES_FILE);
 
+    configure->commitChanges();
```

Comment:
Can we skip this commit, and still have the changed/mapped properties appear as part of the manifest?

It feels wrong to write the config right after reading it, and we will do the mapping of the values anyway while reading. I think persisting them in a batch with the first normal persist is enough, and we can rely on in-memory mapping integers to milliseconds.